database-seeds/collections/tables.json

```diff
@@ -1313,6 +1313,66 @@
 		"tableID": "chunithm-Single-sun-difficulties",
 		"title": "CHUNITHM (SUN) (Difficulties)"
 	},
+	{
+		"default": true,
+		"description": "All songs in DDR A3 Final",
+		"folders": [
+			"F7a0b0620a3ce88806706b97e1c1ea4c2b24565f3f9eaf09047b76c4fd0675333",
+			"F45172302eea4e62cdd9181ec3a2e2f042114f5f31ca6a9d155f5b853c58db4cf",
+			"Fa968c12d64c95d888d9df2564e30264234fba32837994b501d99725f94620abb",
+			"Fa619c4bde06f0ecb5af48af851705e575f95a4f78b83285a715ec7adae4533c6",
+			"F7c7cb196126527c42e2cbbf9862472a273bd037a8e22c0e922bc132517b8ba7e",
+			"F38a78a759678cb9906643b1f718c830f46934b887e7513fc119c25ee4b23298b",
+			"Fe90c9520bdd03d08714cd1afce18f1210dda95a03066af5571e8c214d22549cd",
+			"F414aa7a9818769f77194ef90ea6f6baa396f0de7a8adfcdc227cd099534895a5",
+			"F568470ba3c3a6c4527fc5c8947b62c24470ccd51e651e09a8b9620b74ac7e915",
+			"F333f171c6198829df6c96dc0495340c6bef39d931c6960f999b500719b57f3c5",
+			"Fe9965264ce5da0bbb3edf92e086a96acbc64ac1dfbc447ba7c8f8a691afa23f7",
+			"F1a1e4b4dbfcba8cace611994ff7b72af00e2ce54c29c47958441cdcc322ae584",
+			"F1a13102b49fab06903895b41516d19ea5667d68fbedd6dbbc7e5c600ee3264c7",
+			"F38f74ba481c71d4fb3104d8a12ae9d52f9bec4320bdfb8a65efd724227678346",
+			"F7f407f8f36a493113eb0706fdf503544788f679487f78736ebeba027bb728817",
+			"F9620d888c5e84410489974836735a3dd1ccc19cdcf654e181cbaf5a7724df13c",
+			"F4471e173b60fcd17eb4687502d469307bf230b24adb279f5307ede833f8e7969",
+			"F54bb122610a1d009ca1ef7bd66d9c0344fcd1a6be26cf0dcf3b969837158f568",
+			"F55b8d25fbf443999685648f038a2781c93ad86de48ed9563a9a170a701e3d1b1"
+		],
+		"game": "ddr",
+		"inactive": false,
+		"playtype": "DP",
+		"tableID": "ddr-DP-normalpddra3final",
```

Comment:
this ID is weird. It can just be `ddr-DP-a3` (same for sp, can just be `ddr-DP-a3`.)